generators/client/templates/angular/src/main/webapp/app/admin/admin.module.ts.ejs

```diff
@@ -57,7 +57,7 @@ import {
         RouterModule.forChild(adminState)
     ],
     declarations: [
-        <%_ if (databaseType !== 'no' && databaseType !== 'cassandra') { _%>
+        <%_ if ((databaseType !== || authenticationType === 'uaa') && databaseType !== 'cassandra') { _%>
```

Comment:
I don't understand the `!== ||` part... Does that work ?

Reply:
No, I forgot the 'no'...